nautobot/core/views/mixins.py

```diff
@@ -903,3 +914,33 @@ def perform_bulk_update(self, request, **kwargs):  # pylint: disable=arguments-d
             return redirect(self.get_return_url(request))
         data.update({"table": table})
         return Response(data)
+
+
+class ObjectChangeLogViewMixin(NautobotViewSetMixin):
+    """
+    UI mixin to list a model's changelog queryset
+    """
+
+    base_template = None
+
+    def changelog(self, request, *args, **kwargs):
+        data = {
+            "base_template": self.base_template,
+            "template": "extras/object_changelog.html",
+        }
+        return Response(data)
+
+
+class ObjectNotesViewMixin(NautobotViewSetMixin):
+    """
+    UI Mixin for an Object's Notes.
+    """
+
+    base_template = None
+
+    def notes(self, request, *args, **kwargs):
+        data = {
+            "base_template": self.base_template,
+            "template": "extras/object_notes.html",
```

Comment:
Not blocking and def a nitpick, but can we use the generic `get_template_name()` here instead of hardcoding the template name?